### PR TITLE
Initial contribution of ZoneMinder Binding

### PR DESCRIPTION
**Pull request by Mr-Eskildsen**
This Pull Request replaces PR #1376 (https://github.com/openhab/openhab2-addons/pull/1376).

### CODE REVIEW

File: addons/binding/org.openhab.binding.zoneminder/ESH-INF/config/zoneminderserver.xml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+			<context>refreshConfig</context>
+			<label>Refresh settings</label>
+		</parameter-group>
+		
+		<parameter-group name="advancedSettings">
+            <context>advancedSettings</context>
+            <label>Advanced</label>
+        </parameter-group>
+        
+		<parameter name="hostname" type="text" required="true" groupName="basic">
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.zoneminder/ESH-INF/config/zoneminderserver.xml]: could you run the code formatter? Here is a tabs/spaces mix

File: addons/binding/org.openhab.binding.zoneminder/readme.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+###Things configuration###
+
+```
+Bridge zoneminder:server:ZoneMinderSample [ hostname="192.168.1.55", user="<USERNAME>", password="<PASSWORD>", telnet_port=6802, refresh_interval_disk_usage=1 ]
+{
+	Thing monitor monitor_1 [ monitorId=1, monitorTriggerTimeout=120, monitorEventText="Trigger activated from OpenHAB" ]
+}
+
+```
+###Items configuration###
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.zoneminder/readme.md]: please add an empty line above here

File: addons/binding/org.openhab.binding.zoneminder/readme.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+				Text 		item=zmMonitor1_FrameState
+			}
+		}
+		Frame label="Monitor Helpers" {
+			Switch item=zmMonitor1_Mode
+		}
+	}
+}
+```
+##Troubleshooting##
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.zoneminder/readme.md]: please add an empty line above here

File: addons/binding/pom.xml [hunk 4]
```diff
@@ -68,6 +68,7 @@
     <module>org.openhab.binding.urtsi</module>
     <module>org.openhab.binding.vitotronic</module>
     <module>org.openhab.binding.yamahareceiver</module>
+	<module>org.openhab.binding.zoneminder</module>
```
Comment (kaikreuzer) [inline addons/binding/pom.xml]: please replace tab by spaces

**Comment by Mr-Eskildsen** [comment]
Thanks @kaikreuzer  for your comments. I believe that I have fixed the issues you have pinpointed. Besides from that I found another issue in the .classpath file - I have not added the library I use in that. I verified how it should look like in the chromecast binding, so hopefully I got that one right.

**Comment by kaikreuzer** [comment]
Thanks, looks good!
Just noticed one missing piece when I wanted to merge, sorry: You have not [signed-off](http://docs.openhab.org/developers/contributing/contributing#sign-your-work) your commits, which we need for contributions, so that you formally certify the origin of the code. Best would be to squash your three commits (`git rebase -i HEAD~3`) and add the Signed-off-by line in the resulting commit.

**Comment by Mr-Eskildsen** [comment]
@kaikreuzer Sorry that I not catched that myself.  Thanks for your advice regarding Rebasing :-), hopefully everything  is as expected.

**Comment by kaikreuzer** [comment]
Well done, thanks :-)

**Pull request merged**